cocotb/share/lib/simulator/simulatormodule.c

```diff
@@ -648,15 +648,14 @@ static PyObject *get_signal_val_long(PyObject *self, PyObject *args)
     return retval;
 }
 
-
 static PyObject *set_signal_val_str(PyObject *self, PyObject *args)
```

Comment:
Is this used for both `binstr` and `str`? I remember a change to separate those two paths. Is it currently still in an open PR>

Reply:
Yeah that was with the Questa fixes. #1256. We talked about this in a comment there.

Reply:
I could break that commit out into a separate PR, I'd like to expand it to just not use overloading on `set_signal_value`, which obligates the need for the `using` statements to prevent hiding warnings.

Reply:
I think we want the change adding `gpi_set_signal_value_binstr` that's in that PR before this PR - either by finding time to review that PR in its entirety, or cherry-picking the single commit in question into a standalone PR.

Reply:
You have reviewed that PR. Opening this PR was the one outstanding issue. It probably deserves a few testers since it's FLI.